Evolutionary_loop/Loop_Parts/Part_B/Part_B_PUEO_job2.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+
+########    XF Simulation Software (B)     ########################################################################################## 
+#
+#
+#     1. Prepares output.xmacro with generic parameters such as :: 
```

Comment:
```suggestion
#     1. Prepares output.xmacro with generic parameters such as: 
```

Reply:
Just curious, what is the benefit of doing a :: ? I see it in some of the scripts written by Julie.

Regarding changes, I will leave the PUEO files alone, since I'm not exactly a member of that project.

Reply:
I suggested this change just because it looked inconsistent with the next item in the list, that uses a single `:`.
I'm not aware of any special meaning in this context, especially since it's just a comment in a bash script, but in rst (a markup language commonly used for Python documentation), `::` is used as a shortcut to introduce a code block.

Ok about not wanting to change it, I just point out everything that look wrong/inconsistent to me, but eventually it's up to you to decide.

Reply:
Got it, thanks for the info!